### PR TITLE
refactor(quarks): Sprint Docs 10 Quark maintenance.

### PR DESCRIPTION
**Pull request by 100stacks**
**Sprint Docs 10 Quarks Maintenance.**

- [x] Dev LGTM
- [ ] Dev LGTM

---
Review and capture all quarks added in `Docs 10` to  `src/quarks/quarks.module.js`.

### CODE REVIEW

**Comment by 100stacks** [comment]
Merge after:

- PR https://github.com/rackerlabs/encore-ui/pull/1474
- PR https://github.com/rackerlabs/encore-ui/pull/1498
- PR https://github.com/rackerlabs/encore-ui/pull/1504

File: src/quarks/quarks.module.js [hunk 1]
```diff
@@ -36,6 +38,12 @@
  * * {@link quarks.service:rxAutoSave rxAutoSave}
  * * {@link quarks.service:rxDOMHelper rxDOMHelper}
  * * {@link quarks.service:rxScreenshotSvc rxScreenshotSvc}
+ * * {@link quarks.service:rxBulkSelectUtils rxBulkSelectUtils}
+ * * {@link quarks.service:NotifyProperties NotifyProperties}
+ * * {@link quarks.service:rxFormUtils rxFormUtils}
+ * * {@link quarks.service:routesCdnPath routesCdnPath}
+ * * {@link quarks.service:rxPageTitle rxPageTitle}
+ * * {@link quarks.service:rxNotify rxNotify}
```
Comment (CITguy) [inline src/quarks/quarks.module.js]: Please alphabetize filters

**Comment by 100stacks** [comment]
@CITguy, sorted alphabetically :+1:

**Comment by CITguy** [comment]
LGTM

**Comment by CITguy** [comment]
Merging. Purely documentation with no errors with grunt task.

screen shot 2016-01-21 at 10 40 19 am

**Pull request merged**